README.md

```diff
@@ -20,21 +20,25 @@ MONAI is a [PyTorch](https://pytorch.org/)-based, [open-source](https://github.c
 - multi-GPU data parallelism support.
 
 ## Installation
-Clone and build this repository from source
-
-```bash
-git clone https://github.com/Project-MONAI/MONAI.git
-pip install -e MONAI/
-```
-
-Run some of the examples in [Getting Started](##getting-started)
+Clone and build this repository from source:
+  ```bash
+  git clone https://github.com/Project-MONAI/MONAI.git
+  pip install -e MONAI/
+  ```
+
+Alternatively, pre-built Docker image is available via [DockerHub](https://hub.docker.com/r/projectmonai/monai):
+  ```batch
+  # with docker v19.03+
+  docker run --gpus all --rm -ti --ipc=host projectmonai/monai:latest
```

Comment:
For GPU features, suggest `docker run --runtime=nvidia ...` according to the NGC document.